docs/articles/expensify-classic/workspace-and-domain-settings/Categories.md

```diff
@@ -1,5 +1,137 @@
 ---
 title: Categories
-description: Categories
+description: Categories are used to classify and organize expenses
 ---
-## Resource Coming Soon!
+# Overview
+Categories are commonly used to classify and organize expenses for both personal finances and business accounting.
```

Comment:
Add new line after sentence to space it out so it complys with the [doc](https://docs.google.com/document/d/17Ax1tPL2JkAx8wm6V0QlMUISLN5x_kwdmRsLcv2UgZw/edit) copy.

e.g:
Categories are commonly used to classify and organize expenses for both personal finances and business accounting.

The way Categories function will vary depending on whether or not you connect Expensify to a direct account integration (i.e., QuickBooks Online, NetSuite, etc.).

Reply:
@sarajojacobson there's still a new line missing after "for both personal finances and business accounting."